es/index.js

```diff
@@ -187,3 +215,5 @@ axiosRetry.isNetworkError = isNetworkError;
 axiosRetry.isSafeRequestError = isSafeRequestError;
 axiosRetry.isIdempotentRequestError = isIdempotentRequestError;
 axiosRetry.isNetworkOrIdempotentRequestError = isNetworkOrIdempotentRequestError;
+axiosRetry.noDelay = noDelay;
```

Comment:
This is just a function that returns 0, which I don't think it's necessary to expose.